src/Libraries/RevitNodes/Elements/Parameter.cs

```diff
@@ -1,19 +1,24 @@
 ﻿using Autodesk.Revit.DB;
+using RevitServices.Transactions;
+using System.Collections.Generic;
 
 namespace Revit.Elements
 {
     public class Parameter
```

Comment:
It seems better to have some base classes. If there is not a proper existing one, then maybe IDisposable, IFormattable.